introduction/from_flat_php_to_symfony.rst

```diff
@@ -243,8 +243,7 @@ the ``templates/layout.php``:
 You now have a setup that will allow you to reuse the layout.
 Unfortunately, to accomplish this, you're forced to use a few ugly
 PHP functions (``ob_start()``, ``ob_get_clean()``) in the template. Symfony
-solves this using a :doc:`Templating </components/templating>` component.
-You'll see it in action shortly.
+solves this using a `Templating`_ component. You'll see it in action shortly.
```

Comment:
I know this sentence is no longer true. However, I consider updating it out-of-scope. This whole article is also in need for an update (it's written for Symfony 2 - lot's has changed since then).